everware/spawner.py

```diff
@@ -280,17 +278,19 @@ def start(self, image=None):
         except Exception as e:
```

Comment:
Actually I pass on all the exceptions occurred. I agree it's better to use several catch blocks, will fix it